internal/kubernetes/operator/project/project.go

```diff
@@ -198,6 +200,18 @@ func BuildAtlasProject(br *AtlasProjectBuildRequest) (*AtlasProjectResult, error
 		result.Teams = teams
 	}
 
+	if br.Validator.FeatureExist(features.ResourceAtlasProject, featureBCP) {
+		var bcpRef akov2common.ResourceRefNamespaced
+		bcp, err := buildBCP(br.ProjectStore, br.Project.Name, br.ProjectID, br.TargetNamespace, br.Version, br.Dictionary)
+		if err != nil {
+			return nil, err
+		}
+		bcpRef.Name = bcp.Name
+		bcpRef.Namespace = bcp.Namespace
+		result.BCP = bcp
+		projectResult.Spec.BackupCompliancePolicyRef = &bcpRef
+	}
+
```

Comment:
cc @filipcirtog we might need something like this for #3100 ?

Reply:
Yes! Great idea! Just seen that this Feature validator is also used for the data federation export!Thank you!